src/parser/inc/MatrixNode.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <llvm/ADT/APFloat.h>
+#include <llvm/ADT/APInt.h>
+
+#include <vector>
+
+#include "ExprASTNode.hpp"
+#include "LiteralNode.hpp"
+#include "Type.hpp"
+
+namespace AST {
+class MatrixNode : public ExprNode {
```

Comment:
Hmm. Unfortunately, git hasn't flaged this as a file rename, so anyone relying on changes to this file need to be aware of this at merge time...